ursula_cli/shell.py

```diff
@@ -192,10 +192,13 @@ def _run_heat(args, hot):
         LOG.error("You must have python-heatclient in your python path")
         raise Exception(e)
 
+    tenant_var = "OS_TENANT_NAME"
+    if 'OS_TENANT_NAME' not in os.environ:
+        tenant_var = 'OS_PROJECT_NAME'
     CREDS = {
         'username': os.environ['OS_USERNAME'],
         'password': os.environ['OS_PASSWORD'],
-        'tenant_name': os.environ['OS_TENANT_NAME'],
+        'tenant_name': os.environ[tenant_var],
```

Comment:
That will  blow up if you have OS_TENANT_NAME defined and you do not have OS_PROJECT_NAME defined. You need to do a os.environ.get on OS_PROJECT_NAME as well so it won't throw a key error. 

We are assuming all the variables exist in the creds dict, if they do not you will get a key error. I agree that we should do a follow up PR to validate the existence of those envVars. I did it this way to follow what was already done.